test/suites/test_interval.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import sys
+import unittest
+import msgpack
+import warnings
+import tarantool
+import pandas
+import pytz
+
+from tarantool.msgpack_ext.packer import default as packer_default
+from tarantool.msgpack_ext.unpacker import ext_hook as unpacker_ext_hook
+
+from .lib.tarantool_server import TarantoolServer
+from .lib.skip import skip_or_run_datetime_test
+from tarantool.error import MsgpackError
+
+class TestSuite_Interval(unittest.TestCase):
```

Comment:
We don't test Add/Sub in the file, just encoding/decoding.

Reply:
Seems like a part of https://github.com/tarantool/tarantool-python/pull/230#discussion_r983314441 discussion